### ISSUE TITLE
[$40][MSFT-113] Member registration page: Errors and alerts are not being announced for screen reader users.

### ISSUE THREAD

**Issue by veshu** [topcoder-platform/community-app#2814]
**Steps to Reproduce**
1. Go to the Members registration page:https://accounts.topcoder.com/member/registration. 2. Ensure that the screen reader is switched on. 3. Press tab key till your arrive at the user name field' 4. Type a single character in the field. 6. Since one of the criteria for user name is to have a user name between 2-15 characters, an error is displayed. 7. This error is not announced by the screen readers.

**Expected Result**
Screen reader should immediately announce the error message so that the user corrects the field before proceeding.

**Actual Result**
Errors and alerts are not being announced by the screen readers. Screen readers are silent hence users cannot understand the activity on the screen.

**Device**: Desktop/Labtop
**Operating System**: Windows
**Browser**: Chrome

**WCAG Category**: 3. Understandable
**WCAG Level**: A
**WCAG Success Criterion**: 3.3.1 Error Identification
**Screen Reader Used**: JAWS
**Accessibility Audit Tool used**: N/A

**Attachment**
https://topcodermsft-my.sharepoint.com/personal/pd-topcoder_topcodermsft_onmicrosoft_com/Documents/Forms/All.aspx?cid=088f5f5d%2D9b1c%2D4b7e%2Dacbf%2D320c257c9a9a&FolderCTID=0x0120005C598A51049FC14CBA882E1AEE168F51&id=%2Fpersonal%2Fpd%2Dtopcoder%5Ftopcodermsft%5Fonmicrosoft%5Fcom%2FDocuments%2FMSFT%2DTeams%2DQA%2FJune%202019%2F29%20Jun%2FTest%20Results%2FBug%20Videos%20%26%20Log%20Files%2FKameshwarid%2Fbug%2018

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096270 has been created for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096270 has been updated - the new changes has been updated for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096270 has been updated - it has been assigned to rohitgupta_.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by r0hit-gupta** [comment]
PR at https://github.com/appirio-tech/accounts-app/pull/247

**Comment by cwdcwd** [comment]
[500]: Challenge for the updated issue 2814 is creating, rescheduling this event<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[400]: Failed to create challenge. Detail: Operation failed in the contest service facade.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[400]: Failed to create challenge. Detail: Operation failed in the contest service facade.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[500]: Challenge for the updated issue 2814 is creating, rescheduling this event<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[400]: Failed to create challenge. Detail: Operation failed in the contest service facade.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[400]: Failed to create challenge. Detail: Operation failed in the contest service facade.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[500]: Challenge for the updated issue 2814 is creating, rescheduling this event<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[500]: Challenge for the updated issue 2814 is creating, rescheduling this event<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30097267 has been created for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30097267 has been updated - it has been assigned to rohitgupta_.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by lakshmiathreya** [comment]
Errors are not heard in Mac/Chrome Voice over. @drasticdpk - pls check on Windows.

**Comment by drasticdpk** [comment]
Errors are not heard Windows/JAWS. @nkumar-topcoder please confirm it was deployed on production correctly. 
https://monosnap.com/file/BkzrjKFWAXGadZ2TZOPyvpV23Du3D8

**Comment by nkumar-topcoder** [comment]
> Errors are not heard Windows/JAWS. @nkumar-topcoder please confirm it was deployed on production correctly.
> https://monosnap.com/file/BkzrjKFWAXGadZ2TZOPyvpV23Du3D8

@drasticdpk could be cache, i have refreshed can you pls reverify ?

**Comment by nkumar-topcoder** [comment]
@mishacucicea this is not working on edge browser. Can you reassign please.

**Comment by mishacucicea** [comment]
@r0hit-gupta Please try to fix the issue on Edge.

**Comment by r0hit-gupta** [comment]
@mishacucicea looking into the issues.

**Comment by Oanh-and-only-Oanh** [comment]
@r0hit-gupta, what's the status on this?

**Comment by PrakashDurlabhji** [comment]
@veshu @codeMinter isnt that a rule that new issues cannot be picked up unless we solve previous feedbacks?
or is it allowed for few peoples?



I do not have any problem particularly but it disappoints

**Comment by PrakashDurlabhji** [comment]
@Oanh-and-only-Oanh and I commented simultaneously. what a timing.

**Comment by PrakashDurlabhji** [comment]
@codeMinter can you please say something on my above comment? It hurts when hard work is taken directly like this. If he would have taken anyother open issues , then it wont be a problem but it is surely not a good thing I feel.

**Comment by nkumar-topcoder** [comment]
@mishacucicea @r0hit-gupta can you pls let me know if this issue is now only on Edge but working on chrome ?

**Comment by r0hit-gupta** [comment]
@nkumar-topcoder @mishacucicea working for me on Mac/Chrome.

**Comment by r0hit-gupta** [comment]
@codeMinter @veshu @PrakashDurlabhji I really do not feel I have taken over your work wrongfully. The issue was marked as open for pickup and I simply raised a PR against it. I agree that I missed this feedback which could have been addressed earlier but I believe to err is human. 
I have been resolving all of my feedbacks before proceeding with other issues. It's just this one issue which got left out. 

I am confident enough that @veshu @codeMinter can support my statements here as they have been overlooking my PRs and issues from the beginning.

@PrakashDurlabhji still I apologise if you feel I have done you wrong. I had no intentions of it.

**Comment by r0hit-gupta** [comment]
@nkumar-topcoder @mishacucicea @Oanh-and-only-Oanh for me, errors are being read on both Mac/Chrome using VoiceOver & Windows/Edge using JAWS. 
Test URL - https://accounts.topcoder-dev.com/member/registration
Please check if the page isn't cached. 

P.S. Topcoder cache management surely has some issues. Even on the production env, I have to refresh several times to see the latest challenges and challenge details. Just letting you guys know as a feedback. Thanks!

**Comment by ajefts** [comment]
@Oanh-and-only-Oanh I did a test of this.  It does announce the error message now that is below the username input field:

![image](https://user-images.githubusercontent.com/3812270/62092868-f2a60e80-b244-11e9-9dd5-85fbcb7ca40e.png)


I'm not sure if this is being covered elsewhere, but we should also announce the tooltip for username format so the user knows what the allowed format is.

**Comment by r0hit-gupta** [comment]
@ajefts @Oanh-and-only-Oanh tooltips are handled in issue https://github.com/topcoder-platform/community-app/issues/2813

**Comment by drasticdpk** [comment]
@ajefts   now it is working as expected.
here is the reference link 
https://monosnap.com/file/CUuS1OJbs6LoDUefcrzf5LAoqELBuq

**Comment by crazyk07** [comment]
Payment task has been updated: https://software.topcoder.com/review/actions/ViewProjectDetails?pid=30097267<br/><br/>```This is an automated message for crazyk via Topcoder X```
